inference-engine/src/transformations/src/transformations/op_conversions/convert_nms_to_nms_ie_internal.cpp

```diff
@@ -103,14 +104,14 @@ ngraph::pass::ConvertNMSToNMSIEInternal::ConvertNMSToNMSIEInternal() {
         Output<Node> output_0 = nms_legacy->output(0);
         if (nms_5->output(0).get_element_type() != output_0.get_element_type()) {
             output_0 = std::make_shared<opset1::Convert>(output_0, nms_5->output(0).get_element_type());
-            output_0.get_node_shared_ptr()->set_friendly_name(nms_5->get_friendly_name() + "/convert.0");
+            output_0.get_node_shared_ptr()->set_friendly_name(op::util::create_ie_output_name(nms_5->output(0)));
```

Comment:
@EgorDuplensky Could you add new tests?

Reply:
there is the test for this particular transformation.
But in this case we should have subgraph test (which includes inference), right?

Reply:
Yes right.

Reply:
Done